examples/examples/discovery/README.md

```diff
@@ -0,0 +1,34 @@
+# Discovery
+
+## Running The Example
+
+This example demonstrates how to list all active services within your Elkodon
+application. To get the most out of this example, you can combine it with other
+examples, such as the [event example](../event/) or the
+[publish-subscribe example](../publish_subscribe/), to see active services in
+action.
+
+To begin, let's start some interesting services. Open two terminals and execute
+the following commands:
+
+**Terminal 1**
+
+```sh
+cargo run --example event_listener
```

Comment:
This terminates with `Strg+C` do I need a keyboard with `Ctrl+C` :sweat_smile:

Reply:
Yes please ... join the dark-side, we have an english layout :smile: